lms/djangoapps/commerce/models.py

```diff
@@ -35,9 +36,16 @@ class Meta(object):
     )
     receipt_page = models.CharField(
         max_length=255,
-        default='/commerce/checkout/receipt/?orderNum=',
+        default='/checkout/receipt/?orderNum=',
         help_text=_('Path to order receipt page.')
     )
+    site = models.ForeignKey(
```

Comment:
Leave commerce configuration the way it is.  Create new fields in the site config model.